master/static/srv/proto_get_trials_plus.sql

```diff
@@ -144,6 +145,7 @@ SELECT
   t.metadata as metadata
 FROM searcher_info
   INNER JOIN trials t ON t.id = searcher_info.trial_id
+  LEFT JOIN runs r ON t.id = r.id
```

Comment:
Maybe we can select `log_signal` in the [trials](https://github.com/determined-ai/determined/blob/924f663750963df5ae2fbdb4c284579acfe370ff/master/static/views_and_triggers/up/metrics.sql#L2) view, so we can reduce another join here.